app/javascript/components/upload/DifferentialExpressionStep.jsx

```diff
@@ -24,6 +24,267 @@ export default {
+          </tbody>
+        </table>
+        <p>
+          Seurat can output one-vs-rest DE via <a href="https://satijalab.org/seurat/reference/findallmarkers" target="_blank"><code>FindAllMarkers</code></a>,
+          which can be combined with pairwise DE output from <a href="https://satijalab.org/seurat/articles/de_vignette" target="_blank"><code>FindMarkers(..., ident.1="A", ident.2="B")</code></a>.
+        </p>
+      </>
+    )
+  } else {
+    return (
+      <>
+        <table className="table-terra de-example">
+          <thead>
+            <tr><td>{headers['gene']}</td><td>{headers['group']}</td><td>comparison_group</td><td className="orange">{headers['size']}</td><td className="pink">{headers['significance']}</td><td className="optional">mean</td><td>...</td></tr>
+          </thead>
+          <tbody>
+            <tr><td>It2ma</td><td className="blue">A</td><td className="red">rest</td><td>0.00049</td><td>0.00009</td><td>6.00312</td><td>...</td></tr>
+            <tr><td>Sergef</td><td className="blue">A</td><td className="red">rest</td><td>-0.00036</td><td>0.00239</td><td>4.20466</td><td>...</td></tr>
+            <tr><td>...</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+            <tr><td>It2ma</td><td className="green">B</td><td className="red">rest</td><td>-3.00246</td><td>0.00000</td><td>0.51128</td><td>...</td></tr>
+            <tr><td>Sergef</td><td className="green">B</td><td className="red">rest</td><td>0.00036</td><td>0.074825</td><td>12.71389</td><td>...</td></tr>
+            <tr><td>...</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+            <tr><td>It2ma</td><td className="blue">A</td><td className="green">B</td><td>-0.10246</td><td>0.40019</td><td>0.41357</td><td>...</td></tr>
+            <tr><td>Sergef</td><td className="blue">A</td><td className="green">B</td><td>0.00060</td><td>0.00005</td><td>1.82731</td><td>...</td></tr>
+            <tr><td>...</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+            <tr><td>It2ma</td><td className="blue">A</td><td className="yellow">C</td><td>0.00249</td><td>0.00103</td><td>0.42130</td><td>...</td></tr>
+            <tr><td>Sergef</td><td className="blue">A</td><td className="yellow">C</td><td>-0.00049</td><td>0.02648</td><td>1.06551</td><td>...</td></tr>
+            <tr><td>...</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
+          </tbody>
+        </table>
+        { dePackage === 'scanpy' &&
+        <p>Scanpy can output one-vs-rest DE via <a href="https://scanpy-tutorials.readthedocs.io/en/latest/pbmc3k.html#Finding-marker-genes" target="_blank"><code>rank_gene_groups()</code></a>.</p>
+        }
+        {dePackage === 'other' &&
+        <p>The format for uploaded DE files is flexible.  Just "Choose file" and refine headers below.</p>
+        }
+      </>
+    )
+  }
+}
+
+// TODO: Restore if there is interest in wide format
```

Comment:
I should clarify: wide format support was hidden because it's unsupported in the new inferred menus, which means it's effectively no longer supported.  

Given its compactness and ability to represent DE data for a gene for all comparisons in a single row, wide format was a promising idea for at least _internal SCP output_, as it'd economically enable novel exploration features.  But given A) it now requires targeted effort to restore support, B) we support long format, and C) your compelling points earlier this year, I no longer advocate its worthwhileness wide format as an _author-provided input_.  

So I think it'd make sense to restore support for wide format e.g. if we see any individual researchers (beyond NASA GeneLab) using wide format for DE, or if users indicate long format is an issue e.g. when we [advertise author DE](https://broadworkbench.atlassian.net/browse/SCP-5238).